hikari/events/guild_events.py

```diff
@@ -237,6 +238,109 @@ def guild_id(self) -> snowflakes.Snowflake:
         return self.guild.id
 
 
+@attr_extensions.with_copy
+@attr.define(kw_only=True, weakref_slot=False)
+@base_events.requires_intents(intents.Intents.GUILDS)
+class GuildJoinEvent(GuildVisibilityEvent):
+    """Event fired when the bot joins a new guild.
+
+    !!! note
+        Some fields like `members` and `presences` are included here but not on
+        the other `GuildUpdateEvent` and `GuildUnavailableEvent` guild visibility
+        event models.
+    """
+
+    shard: gateway_shard.GatewayShard = attr.field(metadata={attr_extensions.SKIP_DEEP_COPY: True})
+    # <<inherited docstring from ShardEvent>>.
+
+    guild: guilds.GatewayGuild = attr.field()
+    """The guild the bot just joined.
+
+    Returns
+    -------
+    hikari.guilds.Guild
+        The guild that relates to this event.
```

Comment:
Attributes dont need this section

Reply:
Though, other events have it.

Reply:
Probably inherited from the property. Those need to be removed, and will be done so with the switch to `pdoc`, as we will be able to clearly differentiate from a property and an argument in the rendered docs

Reply:
So remove it from this one prematurely? Otherwise, I'll have to remove it from the other events' docstring as well.

Reply:
Removed on https://github.com/hikari-py/hikari/pull/809/commits/51aff3975d5826cc9f404c5b5a5d2d786f89ee0d